### DISCUSSION TITLE
Edit the command line?

### DISCUSSION THREAD

**Discussion by kr0kon** [Foxlider/FASTER#189] [Q&A] [answered]
Almost DEFINITELY a stupid question, but I'm new to arma 3 server tomfoolery, and I'm wondering where I can edit the actual command line itself, manually. I'm fairly sure I missed something in the manager or something like that, but again, I don't know much about this kinda thing.

**Comment by Foxlider** [comment, marked as answer]
The command line is not editable. 
The command line is generated from the settings you put in the UI. Edit the different UI elements on the different tabs. 
If the commandline is incomplete, either file a bug for a missing feature or copy and edit it by hand before using the command prompt to launch Arma

**Reply by kr0kon** [reply]
ahhh, copying it and running it myself makes a lot of sense, should've thought of that lol, thanks!
although, filing a bug might be a good idea for the long term (unless there's a way to add a -servermod with the UI that i missed)

**Reply by OverlordZorn** [reply]
yes, you can add server mods

**Reply by OverlordZorn** [reply]
![grafik](https://github.com/user-attachments/assets/d8cfe61b-3968-4069-8651-5c581e7f1b5c)
